src/components/Creatable/Creatable.js

```diff
@@ -151,9 +156,19 @@ function CreatableComponent({
               placeholder={keyPlaceholder}
               type={type}
               value={keyValue}
+              className={"testing-a"}
+              style={{
+                marginBottom: hasBothHelperText || keyHelperText ? "0" : valueHelperText ? "1.5rem" : "0rem",
```

Comment:
Is this to account for each input potentially having a combination of label and/or helper text or neither?

Reply:
Oh I need to remove this test class
Yes, this is to maintain the input/button position with different helperText/label combinations
There should be a better way to do this, but I couldn't think in any other solution at the moment.